test/js/web/websocket/autobahn.test.ts

```diff
@@ -175,3 +175,4 @@ if (isDockerEnabled() && (await load())) {
 } else {
   it.todo("Autobahn WebSocket not detected");
 }
+// last test is 13.7.18
```

Comment:
Necessary?

Reply:
running these tests locally is really slow so i felt it was a nice note to myself to know how close to the end i was the next time i or someone is back here

Reply:
what does this comment mean?